perf/lua/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_custom_target(test-lua-perf
+                  DEPENDS 1mops_write_perftest
+                          box_select_perftest
+                          uri_escape_unescape_perftest
+                  COMMENT "Running Lua performance tests"
+)
+
+set(TARANTOOL_BIN ${PROJECT_BINARY_DIR}/src/tarantool)
```

Comment:
You can use genexpr here: `$<TARGET_FILE:tarantool>`. See more info [here](https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#target-dependent-expressions).

Reply:
Fixed, thanks!